tests/network/connectivity/conftest.py

```diff
@@ -27,9 +27,9 @@ def vlan_id_3(vlan_index_number):
 
 
 @pytest.fixture()
-def skip_if_not_ipv6_supported_cluster(ipv6_supported_cluster):
+def fail_if_not_ipv6_supported_cluster(ipv6_supported_cluster):
     if not ipv6_supported_cluster:
-        pytest.skip("IPv6 is not supported in this cluster")
+        pytest.fail(reason="IPv6 is not supported in this cluster")
```

Comment:
Lime [my comment in another PR](https://github.com/RedHatQE/openshift-virtualization-tests/pull/405#discussion_r1981081809) - I think that asserting would fit better here.

Reply:
@yossisegev based on Jenia and Vasiliy's response https://github.com/RedHatQE/openshift-virtualization-tests/pull/405/files#r1981288118, I am keeping it as it is.